docs/build-system.md

```diff
@@ -189,19 +187,78 @@ $ node _build/default/app/app.js
 Jane
 ```
 
-One thing to note is that we have to look for the `app.js` file inside an `app`
-folder, but we don’t have any such folder in our sources. The reason why this
-folder is created is to support multiple `melange.emit` stanzas in the same
-folder. To support this scenario, Dune will use the `target` field defined in
-the `melange.emit` to place the artifacts generated from a `melange.emit` stanza
-in the following folder:
+#### JavaScript artifacts layout
+
+One thing to note is that in the command above we had to look for the `app.js`
```

Comment:
> One thing to note is that

remove, not adding to the sentence

Reply:
https://github.com/melange-re/documentation-site/pull/25/commits/3aaaa01f303e8217b4eb543966b527d3b6fd2fbf